PyRDF/Node.py

```diff
@@ -202,10 +214,19 @@ def graph_prune(self):
                 # [If the user had a variable reference to the current node,
                 # the value would be at least 4. Hence nodes with less than
                 # 4 references will have to be removed ]
-
+                #
+                # Python 3:
+                #
+                # Here the minimum number of references a node can have is 1.
+                # In fact, when a user assigns a variable to a node and then
+                # reassigns the same variable to another object, Python 3 
+                # keeps track of past objects assigned to the variable in a list.
```

Comment:
I would not mentioned this particularity here,  we have seen that both Python2 and Python3 keep this history of previous references.